Examples/Swift/ViewController.swift

```diff
@@ -114,18 +121,18 @@ class ViewController: UIViewController, MGLMapViewDelegate, NavigationViewContro
             }
             
             self?.userRoute = route
-            self?.toggleNavigationButton.isHidden = false
+            self?.startNavigationButton.isHidden = false
+            self?.simulateNavigationButton.isHidden = false
             self?.howToBeginLabel.isHidden = true
             
-            
             // Open method for adding and updating the route line
             self?.mapView.showRoute(route)
             
             didFinish?()
         }
     }
     
-    func startNavigation(_ route: Route) {
+    func startNavigation(_ route: Route,_ simulatesLocationUpdates: Bool = false) {
```

Comment:
Let’s follow [Swift argument naming guidelines](https://swift.org/documentation/api-design-guidelines/#argument-labels). Only use `_` for the first argument, and only when the name of the method outside the parentheses already describes the first argument. (In general, we want the call site to include labels like `foo: `.) In this case, “navigation” doesn’t describe the first parameter, and the second argument shouldn’t have `_` either:

```swift
func startNavigation(along route: Route, simulatingLocationUpdates simulatesLocationUpdates: Bool = false)
```

This way, the call site looks like this:

```swift
startNavigation(along: route, simulatingLocationUpdates: true)
```

Someone reading this call site knows what the method will do with `route`, without having to read documentation.